src/plugins/console/public/application/containers/editor/monaco/monaco_editor.tsx

```diff
@@ -39,6 +62,42 @@ export const MonacoEditor = ({ initialTextValue }: EditorProps) => {
         width: 100%;
       `}
     >
+      <EuiFlexGroup
+        className="conApp__editorActions"
+        id="ConAppEditorActions"
+        gutterSize="none"
+        responsive={false}
+        style={editorActionsCss}
+      >
+        <EuiFlexItem>
+          <EuiToolTip
+            content={i18n.translate('console.sendRequestButtonTooltip', {
+              defaultMessage: 'Click to send request',
+            })}
+          >
+            <EuiLink
+              color="success"
+              onClick={sendRequestsCallback}
+              data-test-subj="sendRequestButton"
+              aria-label={i18n.translate('console.sendRequestButtonTooltip', {
+                defaultMessage: 'Click to send request',
+              })}
+            >
+              <EuiIcon type="playFilled" />
+            </EuiLink>
+          </EuiToolTip>
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <ConsoleMenu
+            getCurl={getCurlCallback}
+            getDocumentation={() => {
+              return Promise.resolve(null);
+            }}
+            autoIndent={() => {}}
+            notifications={notifications}
+          />
```

Comment:
Is the current "Open documentation" action intend to be contextual to the currently highlighted command? If not, would both "Open documentation" and "Auto indent" actions be better served as global actions rather than individual command actions?